app/models/book.rb

```diff
@@ -91,6 +91,11 @@ class Book < ActiveRecord::Base
   default_scope { where(in_store: true) }
 
   CSV_COLUMNS = ["Book Name", "ASIN"]
+  # Probably want future improvement so we don't have to store it in a .yml
+  # file which will get parsed every time the model is used. We can also
+  # allow for future mapping changes by making a model mapping search tags
+  # to model tags.
+  LEVELS_CONVERT = YAML.load(File.read(File.expand_path('../../../db/levelsConvert.yml', __FILE__)))
```

Comment:
Line is too long. [100/80]<br>Inconsistent indentation detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.